tests/http/testenv/nghttpx.py

```diff
@@ -226,7 +220,7 @@ def start(self, wait_live=True):
         self._process = subprocess.Popen(args=args, stderr=ngerr)
         if self._process.returncode is not None:
             return False
-        return not wait_live or self.wait_live(timeout=timedelta(seconds=5))
+        return not wait_live or self.wait_live(timeout=timedelta(seconds=30))
```

Comment:
It occurs to me that all these 30 second timeouts should probably be one (or a few) constant(s) defined in one place. That would prevent a future package-wide search-and-replace exercise like this one.

Reply:
I'll make that a constant in Env.